backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -12,7 +12,12 @@ Because all taken backup remains until expired, also check the time to live (TTL
 
 [NOTE]
 ====
-There might be known issues with supported storage classes, for example, CSI backups might fail due to a Ceph limitation. For more information, see xref:../../../backup_and_restore/application_backup_and_restore/release-notes/oadp-release-notes-1-1.adoc#known-issues_oadp-release-notes[Known issues].
+There might be known issues with supported storage classes, for example, CSI backups might fail due to a Ceph limitation.
+
+You can create many snapshots of the same persistent volume claim (PVC) but cannot schedule periodic creation of snapshots:
+
+* For CephFS, you can create up to 100 snapshots per PVC.
+* For RADOS Block Device (RBD), you can create up to 512 snapshots for each PVC. (link:https://issues.redhat.com/browse/OADP-804[*OADP-804*]) and (link:https://issues.redhat.com/browse/OADP-975[*OADP-975*])
```

Comment:
As we would cut 1.1 release notes, i have moved this section over to replace the link

Do we need to keep this?

Reply:
@amastbau - please can I flag this question to you? thanks